app/kuma-dp/pkg/dataplane/envoy/remote_bootstrap.go

```diff
@@ -8,12 +8,17 @@ import (
 
 	kuma_dp "github.com/Kong/kuma/pkg/config/app/kuma-dp"
 	util_proto "github.com/Kong/kuma/pkg/util/proto"
-	xds_bootstrap "github.com/Kong/kuma/pkg/xds/bootstrap"
 	envoy_bootstrap "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v2"
 	"github.com/gogo/protobuf/proto"
 	"github.com/pkg/errors"
 )
 
+type BootstrapRequest struct {
```

Comment:
Instead of duplicating this type, could we just move it into another package ?

Something like `github.com/Kong/kuma/pkg/xds/bootstrap/api` or `github.com/Kong/kuma/pkg/xds/bootstrap/rest`